ci-operator/step-registry/dpf/hypervisor/load-kubeconfig/dpf-hypervisor-load-kubeconfig-commands.sh

```
@@ -75,3 +75,33 @@ oc --kubeconfig=/tmp/kubeconfig.doca8 config unset "clusters.${CLUSTER_NAME}.cer
 
 cp /tmp/kubeconfig.doca8 "${SHARED_DIR}/kubeconfig"
 echo "Kubeconfig copied to \${SHARED_DIR}/kubeconfig successfully"
+
+# The *.apps wildcard DNS is internal to the hypervisor network and not
+# resolvable from CI pods. Resolve the ingress VIP from the hypervisor
+# and write /etc/hosts entries via proxy-conf.sh so downstream steps
+# pick them up automatically.
+APPS_DOMAIN="$(oc --kubeconfig=/tmp/kubeconfig.doca8 get ingresses.config.openshift.io cluster -o jsonpath='{.spec.domain}')"
+echo "Resolving apps domain '*.${APPS_DOMAIN}' from the hypervisor..."
+APPS_HOSTNAME="console-openshift-console.${APPS_DOMAIN}"
+APPS_IP="$(ssh ${SSH_OPTS} root@${REMOTE_HOST} "getent hosts ${APPS_HOSTNAME} | awk '{print \$1}'")"
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)